### PR TITLE
Expect ModelGrade.Code to be measured

### PR DESCRIPTION
**Pull request by pedronis**
asserts.ModelGrade is growing a Code method to give us a fixed size
bit representation convenient for measuring, expect the use of that

drive-by: make the series values in the tests more realistic

### CODE REVIEW

**Comment by pedronis** [comment]
Requires https://github.com/snapcore/snapd/pull/8522

**Comment by pedronis** [comment]
@chrisccoulson the snapd bits have landed and now this is green as well

File: snapmodel_policy.go [hunk 1]
```diff
@@ -80,12 +57,8 @@ type SnapModelProfileParams struct {
 //  digest2 = H(digest1 || model)
 //  digestModel = H(digest2 || series || grade)
 // The signing key digest algorithm is encoded in little-endian format, and the sign-key-sha3-384 field is hashed in decoded (binary)
-// form. The brand-id, model and series fields are hashed without null terminators. The grade field is encoded as a single byte with
-// the following conversion:
-//  "unset":     0
-//  "secured":   1
-//  "signed":    2
-//  "dangerous": 3
+// form. The brand-id, model and series fields are hashed without null terminators. The grade field is encoded as the 32 bits from asserts.ModelGrade.Code.
```
Comment (chrisccoulson) [inline snapmodel_policy.go:60]: I would be explicit that the 4-byte grade field is encoded in little endian.

Comment (pedronis) [reply snapmodel_policy.go]: done

**Pull request merged**